server.js

```diff
@@ -9,12 +9,11 @@ var secrets = require('./secrets');
 var axios = require('axios');
 
 var app = express();
-var router = express.Router();
```

Comment:
From my perspective the problem was in version of node. So, you don't need to remove router.